app/frontend/styles/_feedback.scss

```diff
@@ -0,0 +1,122 @@
+.gem-c-feedback {
```

Comment:
this file contains all relevant classes from https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss, unaltered where possible to make any future work using that repository easier